lesson_05/homework_05.py

```diff
@@ -1,39 +1,77 @@
 small_list = [3, 1, 4, 5, 2, 5, 3]
 big_list = [3, 5, -2, -1, -3, 0, 1, 4, 5, 2]
 # task 1. Знайдіть всі унікальні елементи в списку small_list
+print("Всі унікальні елементи в списку small_list:", set(small_list))
 
 # task 2. Знайдіть середнє арифметичне всіх елементів у списку small_list
+average_value = sum(small_list) / len(small_list) # или sum(small_list) // len(small_list) чтоб без десятичных
+print("Cереднє арифметичне всіх елементів у списку small_list:", average_value)
 
 # task 3. Перевірте, чи є в списку big_list дублікати
-
+length_list = (len(big_list))
+unique_values = set(big_list)
+duplicates = (length_list == unique_values)
+if duplicates != True :
```

Comment:
тут можна спростити і додати правильний вивід
```suggestion
if duplicates:
```